test/html/expect/test_package+custom_theme,ml/Include/index.html

```diff
@@ -19,68 +19,68 @@
   </nav>
   <header>
    <h1>
-    Module <code>Include</code>
+    Module <code><span>Include</span></code>
    </h1>
   </header>
   <div class="content">
    <div class="spec module-type" id="module-type-Not_inlined">
-    <a href="#module-type-Not_inlined" class="anchor"></a><code><span class="keyword">module</span> <span class="keyword">type</span> <a href="module-type-Not_inlined/index.html">Not_inlined</a> = <span class="keyword">sig</span> ... <span class="keyword">end</span></code>
+    <a href="#module-type-Not_inlined" class="anchor"></a><code><span><span class="keyword">module</span> <span class="keyword">type</span> </span><span><a href="module-type-Not_inlined/index.html">Not_inlined</a></span><span> = <span class="keyword">sig</span> ... <span class="keyword">end</span></span><span></span></code>
    </div>
    <div>
     <div class="spec include">
      <div class="doc">
       <details open="open">
        <summary>
-        <span class="def"><code><span class="keyword">include</span> <a href="module-type-Not_inlined/index.html">Not_inlined</a></code></span>
+        <span class="def"><code><span><span class="keyword">include</span> <a href="module-type-Not_inlined/index.html">Not_inlined</a></span></code></span>
        </summary>
        <div class="spec type" id="type-t">
-        <a href="#type-t" class="anchor"></a><code><span class="keyword">type</span> t</code>
+        <a href="#type-t" class="anchor"></a><code><span><span class="keyword">type</span> t</span><span></span><span></span><span></span></code>
```

Comment:
Is it possible to elide those empty spans?

Reply:
Should be done now. maybe even too aggressively (it removes empty code blocks too...)